examples/silicon.jl

```diff
@@ -9,6 +11,8 @@ Si = ElementPsp(:Si, psp=load_psp("hgh/lda/Si-q4"))
 atoms     = [Si, Si]
 positions = [ones(3)/8, -ones(3)/8]
 
-model = model_LDA(lattice, atoms, positions)
+# model = model_LDA(lattice, atoms, positions, temperature=1e-2, smearing=Smearing.MethfesselPaxton(2))
+model = model_LDA(lattice, atoms, positions, temperature=1e-2u"Ry", smearing=Smearing.MarzariVanderbilt())
```

Comment:
What? Don't use smearing for silicon. Also silicon is our hello world, so I would keep this as basic as possible and stick to the defaults

Reply:
Oh that is an error. I was playing around with this file.